### PR TITLE
[add_otel] Add OpenTelemetry tests

### PR DESCRIPTION
**Pull request by jasondlee**
Create image
Update BOM
Create test app
Create integration test

### CODE REVIEW

File: images/opentelemetry/pom.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~  Copyright 2022 Red Hat, Inc., and individual contributors
```
Comment (fabiobrz) [inline images/opentelemetry/pom.xml:4]: Nitpicking, maybe but shouldn't it be 2023? Again, nothing blocking anything, but just for my information, are the headers generated or do we need to copy/paste them manually?

**Review by fabiobrz — APPROVED** [review]
HI @jasondlee - thanks for this PR. CI see this would be a good "smoke" test for MicroProfile Telemetry WildFly integration, and the changes LGTM with respect to that.

File: images/opentelemetry/pom.xml (hunk 1 above)
Comment (kabir) [reply images/opentelemetry/pom.xml:4]: People generally have these set up in their IDE. IntelliJ supports something like $day.year which gets replaced with the current year though :-)

Comment (jasondlee) [reply images/opentelemetry/pom.xml:4]: That was probably a copy-and-paste class. My IDE collapses the header comments, so I missed that. I'll get that updated and verify any others.

**Comment by jasondlee** [comment]
Copyright header updated.

**Pull request merged**